src/app/api/views/comment.py

```diff
@@ -0,0 +1,41 @@
+from rest_framework import views, status
+from rest_framework.response import Response
+from rest_framework.permissions import IsAuthenticated
+
+from app.permissions import OwnProfilePermission
+
+from app.services.comment import CommentService
+from app.domain.repositories.comment import CommentRepository
+from app.api.serializers.comment import CommentSerializer
+
+
+class CommentCreateView(views.APIView):
+    permission_classes = (IsAuthenticated,)
+    service = CommentService(CommentRepository())
+    serializer_class = CommentSerializer
+
```

Comment:
Aqui poderia ficar assim por exemplo

```suggestion
class CommentCreateView(views.APIView):
    permission_classes = (IsAuthenticated,)
    service = CommentService()
    serializer_class = CommentSerializer

```